src/pages/Apps/Apps.scss

```diff
@@ -8,6 +8,7 @@
 
     .app-shortcut {
       position: relative;
```

Comment:
Seems that this can be omitted because of the same property of `.close-btn-boundaries`.

Reply:
Done.